nautobot/core/management/commands/generate_test_data.py

```diff
@@ -83,7 +83,6 @@ def _generate_factory_data(self, seed, db_name):
             from nautobot.extras.utils import TaggableClassesQuery
             from nautobot.ipam.choices import PrefixTypeChoices
             from nautobot.ipam.factory import (
-                IPAddressFactory,
```

Comment:
The IPAddressFactory gets called as a subfactory of PrefixFactory. Should we output the hash for IPAddressFactory at the same time as PrefixFactory?

Reply:
Maybe, but would have to be a special case in the `_create_batch` function, not sure it's worth it?